pgraph/pgraph_test.go

```diff
@@ -934,3 +934,21 @@ func TestSetValue(t *testing.T) {
 		t.Errorf("expecting value of %s at %s position, got %v", value, key, v)
 	}
 }
+
+
+//Dov's new test
+
+//Testing the naming of graphs
+func TestSetName(t *testing.T) {
+	name := "Dov's Graph"
```

Comment:
Since we want to ultimately merge this patch, referring to "Dov" isn't helpful. Usually we only refer to well-known names like "einstein" or "monty python" or leave it generic "graph1" or "foo".